content/author.yaml

```diff
@@ -2,6 +2,10 @@
   bio: LoginRadius is a leading provider of cloud-based Customer Identity and Access Management (cIAM) platform.
   github: "LoginRadius"
 
+- id: Emadamerho-Atori Nefe
+  bio: A web developer and who loves creating technical content and sharing his knowledge with others.
```

Comment:
@nefejames Please add more information to your author bio -- ideally, around 300 characters.

Reply:
Hi @raghunath-r-a I have updated my bio.